ZIGSIMPlus/Views/CommandOutputTabNavigationController.swift

```diff
@@ -12,13 +12,5 @@ class CommandOutputTabNavigationController: UINavigationController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Initialize child view controllers
-        for viewController in viewControllers {
-            if type(of: viewController) == CommandOutputViewController.self {
-                let vc = viewController as! CommandOutputViewController
-                vc.presenter = CommandOutputPresenter(view: vc)
-            }
-        }
     }
```

Comment:
Can we remove these `viewDidLoad` s?

Reply:
Actually they work without viewDidLoad. 
Do you want to remove them?

Reply:
I don't care, but I might remove these code later 😄